### DISCUSSION TITLE
nuxt 3 site is ugly. it look like I designed it

### DISCUSSION THREAD

**Discussion by n4an** [nuxt/nuxt#9933] [General]
colors are bad.

**Comment by kissu** [comment]
Hi, can you please provide some measurable values? Maybe your monitor is poorly calibrated.
Also, which ones do you suggest? There is a dedicated Figma where you can see the design system. It is probably a good starting point to decide on what suits you the best for a Pull Request! 🤩

**Comment by Geminii** [comment]
> 
> 
> colors are bad. 

Can you more detail about it ? Not really constructed for a discussion at the moment

**Comment by christopher4lis** [comment]
Has to be a troll post, site is one of the best looking framework sites I've ever seen.
